backend/typescript/middlewares/validators/camperValidators.ts

```diff
@@ -9,123 +9,144 @@ import {
 
 /* eslint-disable @typescript-eslint/explicit-module-boundary-types */
 /* eslint-disable-next-line import/prefer-default-export */
-export const createCamperDtoValidator = async (
+export const createCampersDtoValidator = async (
   req: Request,
   res: Response,
   next: NextFunction,
 ) => {
-  if (!validatePrimitive(req.body.campSession, "string")) {
-    return res.status(400).send(getApiValidationError("campSession", "string"));
+  let campSession = "";
+  let chargeId = "";
```

Comment:
actually this gives a lint error and it's suggesting to just declare these as const instead during the assigning since they're never reassigned. that also work?

Reply:
oh right, yes that workss